src/types/index.ts

```diff
@@ -1278,6 +1295,16 @@ export interface PortfolioMovement {
   memo?: string;
 }
 
+export type PortfolioMovementNonFungible = {
```

Comment:
```suggestion
export type NFTPortfolioMovement = {
```
or maybe `NonFungibleAssetPortfolioMovement` to be more explicit